docs/tutorials/add-parameter-values.rst

```diff
@@ -16,32 +16,61 @@ At the stage of creating a model, we use :class:`pybamm.Parameter` and :class:`p
 We then create a :class:`ParameterValues` class, using a specific set of parameters, to iterate through the model and replace any :class:`pybamm.Parameter` objects with a :class:`pybamm.Scalar` and any :class:`pybamm.FunctionParameter` objects with a :class:`pybamm.Function`.
 
 For an example of how the parameter values work, see the
-`parameter values notebook <https://github.com/pybamm-team/PyBaMM/blob/master/examples/notebooks/spatial_methods/finite-volumes.ipynb>`_.
+`parameter values notebook <https://github.com/pybamm-team/PyBaMM/blob/master/examples/notebooks/parameter-values.ipynb>`_.
 
 Adding a set of parameters values
 ---------------------------------
 
-Parameter sets should be added as csv files in the appropriate chemistry folder in ``input/parameters/`` (add a new folder if no parameters exist for that chemistry yet).
+There are two ways to add parameter sets:
+
+1. **Complete parameter file**: Parameter sets should be added as csv files in the appropriate chemistry folder in ``input/parameters/`` (add a new folder if no parameters exist for that chemistry yet).
 The expected structure of the csv file is
 
-+------------+------------+-----------+-------------+-------------+
-| Name       | Value      | Units     | Reference   | Notes       |
-+============+============+===========+=============+=============+
-| Example    | 13         | m.s-2     | bloggs2019  | an example  |
-+------------+------------+-----------+-------------+-------------+
++-------------------------+----------------------+-----------------------+-------------+
+| Name [Units]            | Value                | Reference             | Notes       |
++=========================+======================+=======================+=============+
+| Example [m]             | 13                   | bloggs2019            | an example  |
++-------------------------+----------------------+-----------------------+-------------+
 
 Empty lines, and lines starting with ``#``, will be ignored.
 
+2. **Parameters for a single material**: It's possible to add parameters for a single material (e.g. anode) and then re-use existing parameters for the other materials. To do this, add a new subfolder with a ``README.md`` for references and csv file of parameters (e.g. ``input/parameters/lithium-ion/anodes/new_anode_chemistry_Bloggs2019/``). Then this file can be referenced using the ``chemistry`` option in ``ParameterValues``, e.g.
+
+.. code-block:: python
+
+    param = pybamm.ParameterValues(
+        chemistry={
+            "chemistry": "lithium-ion",
+            "cell": "kokam_Marquis2019",
+            "anode": "new_anode_chemistry_Bloggs2019",
+            "separator": "separator_Marquis2019",
+            "cathode": "lico2_Marquis2019",
+            "electrolyte": "lipf6_Marquis2019",
+            "experiment": "1C_discharge_from_full_Marquis2019",
```

Comment:
Maybe there should be a very brief description of the parameters that go into the experiment folder? Though maybe it is clear from looking at the existing parameter sets.

Reply:
I also think that a list of the required parameters in the documentation would be helpful.